src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -32,10 +32,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0-2.final" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" />
     <PackageReference Include="System.Threading.AccessControl" Version="8.0.0" />
     <PackageReference Include="System.Drawing.Common" Version="8.0.0" />
-    <PackageReference Include="JsonSchema.Net" Version="5.2.6" />
+    <PackageReference Include="JsonSchema.Net" Version="5.2.7" />
+    <!-- This section is to force the version of non-direct dependencies -->
+    <PackageReference Include="Json.More.Net" Version="1.9.2" />
```

Comment:
^ Transitive

Reply:
If we didn't add this transitive package in the GA release, why do we need to add it now?